test/pom.xml

```diff
@@ -91,7 +91,7 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>nacos-address</artifactId>
         </dependency>
-
```

Comment:
Please revert this.

Reply:
Thank you for your review, I have been fixed it.